etc/gimp.profile

```diff
@@ -18,6 +16,7 @@ include /etc/firejail/whitelist-var-common.inc
 apparmor
 caps.drop all
 net none
+nodbus
```

Comment:
If I understand this correctly nodbus makes settings immutable here - https://github.com/netblue30/firejail/pull/1840

Reply:
I'll add a comment there. Thanks.